Methods Document Representation (#4)
greenelab/annorxiver_manuscript: content/03.methods.md

### Visualizing the Preprint Landscape

#### Generate Document Representation

> We used the gensim [@raw:rehurek_lrec] package (version 3.8.1) to train a word2vec model, continuous bag of words (CBOW), [@arxiv:1301.3781] over the entire bioRxiv repository. 

Reviewer: ```suggestion
We used gensim [@raw:rehurek_lrec] (version 3.8.1) to train a word2vec continuous bag of words (CBOW) [@arxiv:1301.3781] model over the bioRxiv corpus. 
```

> We trained this model for 20 epochs to generate vectors of 300 dimensions.

Reviewer: ```suggestion
Our neural network architecture had 300 hidden nodes, and we trained this model for 20 epochs.
```

> We set the random seed to be 100 and kept the other parameters as gensim's default settings.

Reviewer: ```suggestion
We set a fixed random seed and otherwise used gensim's default settings.
```

Following training, we generated a document vector for every article within bioRxiv and PubMed Central.

> This document vector is calculated by taking the average of every token present within a given article as well as the word2vec model's vocab list.

Reviewer: ```suggestion
This document vector is calculated by taking the average of every token present within a given article, ignoring those that were absent from the word2vec model.
```
